assets/js/src/newsletter_editor/behaviors/HighlightEditingBehavior.js

```diff
@@ -47,9 +49,25 @@ BL.HighlightEditingBehavior = Marionette.Behavior.extend({
       this.view.removeHighlight();
     }
   },
+  onStartResizing: function onStartResizing() {
+    this.onStartEditing();
+    this.view.triggerMethod('resizeStart');
+  },
+  onStopResizing: function onStopResizing(event) {
+    this.onStopEditing();
+    this.view.triggerMethod('resizeStop', event);
+  },
   onDomRefresh: function onDomRefresh() {
     if (this.isBeingEdited) {
       this.view.addHighlight();
     }
   },
+  onChildviewResizeStart: function onChildviewResizeStart() {
```

Comment:
I would write onChild**V**iewResizeStart to respect the camel case rule.

Reply:
I would too. But it isn't an option :( This naming is required by framework: https://marionettejs.com/docs/master/events.html#event-bubbling